crawl-ref/source/dat/des/altar/ecumenical.des

```diff
@@ -419,3 +419,119 @@ x-----x
 x-----x
 xxb+bxx
 ENDMAP
+
+###############################################################
+# RL Ruins as Ecumenical Alters by Shapermc
+# The ruins of Machu Picchu inspired these sites for ecumenical
+# alters. I am aiming to make them feel attractive on the outside 
+# to explore, and unique and random on the inside. 
+
+NAME:   shapermc_ecumenical_altar_mp_vishouse
+TAGS:   transparent no_monster_gen
+SUBST:  - = --.G
+SUBST:  G = GGG-
+COLOUR: - = brown
+FTILE:  -G = floor_rough_brown
+COLOUR: G = lightgrey
+TILE:   G = dngn_crumbled_column
+: set_feature_name("granite_statue", "broken pillar")
+: ecumenical_altar_setup(_G)
+
+MAP
+.............                                               
+.....xxxxx..x                                               
+..xxx---xG--x                                               
+xx------xxxxx                                               
+x--xxx--x---x                                               
+x--x_x------x                                               
+x--b-x--xx+xx                                               
+x--+-x--x---x                                               
+x--b-x--xGGGx                                               
+xxxxxx.xxxxxx                                               
+.............  
+ENDMAP
+
+
+NAME:   shapermc_ecumenical_altar_mp_highgroup
+TAGS:   transparent no_monster_gen
+SUBST:  - = --.G
+SUBST:  G = GGG-
+COLOUR: - = brown
+FTILE:  -G = floor_rough_brown
+COLOUR: G = lightgrey
+TILE:   G = dngn_crumbled_column
+: set_feature_name("granite_statue", "broken pillar")
+: ecumenical_altar_setup(_G)
+NSUBST: ^ = 4=^. / 4=^..
```

Comment:
I did, I wanted it to feel rare when it happened to avoid uncanny valley as you've said. Also, webs can be annoying to otab through, so I didn't want players getting too hung up. I was hoping that maybe one of those turns into a spider web when spawned, two at max.